flink-core/src/main/java/org/apache/flink/core/fs/FileSystem.java

```diff
@@ -248,6 +248,8 @@
             ImmutableMultimap.<String, String>builder()
                     .put("wasb", "flink-fs-azure-hadoop")
                     .put("wasbs", "flink-fs-azure-hadoop")
+                    .put("abfs", "flink-fs-azure-hadoop")
+                    .put("abfss", "flink-fs-azure-hadoop")
```

Comment:
Thanks for adding these schemes in the Map. Otherwise, it will not allow those schemes to be recognized as supported FileSystems.

Reply:
Just to clarify: Adding the entries will not change anything if everything is setup correctly. However, in case the user forgets to add the fs (as plugin or lib), this list can give a meaningful error message. So thanks for updating it!